metrics-core/src/main/java/software/amazon/swage/metrics/jmx/sensor/DiskUsageSensor.java

```diff
@@ -30,7 +30,7 @@
  */
 public class DiskUsageSensor implements Sensor {
 
-    private static final long K = 1024;
+    private static final long G = (long) (1024 * 1024 * 1024);
```

Comment:
Why spaces after the cast parens?  I find this slightly less clear.

I tend to read casts similar to a method call that forces the value, spaces between calls and params don't read well.

Reply:
They're unary operators, though. https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace

You can make the same argument about `if`.

I just thought it was accidental. I'm okay with undoing formatting changes, except maybe the snake->camel renames.

Reply:
Actually ... I argued against myself, maybe. Unary operators in general shouldn't be surrounded by spaces. All the same, I'll do as you suggested here.